qcodes/instrument_drivers/Keysight/Keysight_E8267D.py

```diff
@@ -27,14 +27,19 @@ class Keysight_E8267D(VisaInstrument):
     def __init__(self, name, address, step_attenuator=False, **kwargs):
         super().__init__(name, address, **kwargs)
 
+        # Only listed most common spellings idealy want a
+        # .upper val for Enum or string
+        on_off_validator = vals.Enum('on', 'On', 'ON',
```

Comment:
this is a great case for using [`create_on_off_val_mapping`](http://qcodes.github.io/Qcodes/examples/Creating%20Instrument%20Drivers.html?highlight=create_on_off#On/Off-parameters) function. please use it :)

Reply:
@astafan8 That would indeed be a good case, but it would make the driver backwards incompatible (with the `create_on_off_val_mapping` the parameter will return a 0, instead of 'Off').

Reply:
damn, indeed. ok, let's not introduce backwards incompatibility.